test/cases/api.sh

```diff
@@ -162,13 +191,32 @@ EOM
   $GCP_CMD --version
 }
 
+function installClientAzure() {
+  if ! hash az; then
+    sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+    echo -e "[azure-cli]
```

Comment:
I've just copy-pasted this from the official docs... I added a link to the docs to the script and left `-e` to prevent confusion as this is just a copy. :)